components/server/src/ome/services/ThumbnailBean.java

```diff
@@ -1005,7 +1005,15 @@ public void createThumbnailsByLongestSideSet(Integer size,
         // process due to the possible unloaded Pixels. If we do not,
         // Pixels will be unloaded and we will hit
         // IllegalStateException's when checking update events.
-        iUpdate.saveArray(toSave.toArray(new Thumbnail[toSave.size()]));
+        try {
+        	iUpdate.saveArray(toSave.toArray(new Thumbnail[toSave.size()]));
+        } catch (Exception e) {
```

Comment:
I catch https://github.com/openmicroscopy/openmicroscopy/commit/ccec2d305624c95b446434434538a5c03813d200#diff-bae9d2b989f3f6e6de49048fa4dcd34dR1009

but then was throwing

 ``` throws serverStackTrace = ome.conditions.InternalException:  Wrapped Exception: (org.hibernate.NonUniqueObjectException):
a different object with the same identifier value was already associated with the session: [ome.model.core.Pixels#692188]```

so I went for Exception to check if that would work at all